packages/contracts-bedrock/deploy/021-SystemDictatorSteps-2.ts

```diff
@@ -123,7 +124,7 @@ const deployFn: DeployFunction = async (hre) => {
   })
 
   // Step 4 transfers ownership of the AddressManager and L1StandardBridge to the ProxyAdmin.
-  await doStep({
+  await doStep(hre, {
```

Comment:
You should be able to remove passing through `hre` now?

Reply:
Good catch fixed.